source/dom/dispatcher.js

```diff
@@ -283,11 +283,12 @@
 	enyo.dispatcher.features.push(
 		function (e) {
 			if ("click" === e.type) {
-				if (e.clientX === 0 && e.clientY === 0) {
+				if (e.clientX === 0 && e.clientY === 0 && e.detail === 0) {
```

Comment:
I'm wondering if this could be further simplified so that we only check `e.detail === 0` here as `e.clientX` and `e.clientY` are always 0 when `e.detail` is 0 as far as I can tell, or even change the conditional to `!e.detail` to fully support IE8 (which doesn't implement this property on `MouseEvent`). But pushed this commit for now so we can discuss.